src/state/wallet/hooks.ts

```diff
@@ -70,7 +70,7 @@ export function useTokenBalancesWithLoadingIndicator(
     'balanceOf',
     [address],
     undefined,
-    100_000
+    100_000_000
```

Comment:
Why this change?
Actually, what's this number for?

Reply:
Shouldn't be here, I need to remove this